packages/hoc/src/withWhiteSpace/__snapshots__/test.js.snap

```diff
@@ -20,6 +20,7 @@ exports[`withWhiteSpace matches wrapper snapshot 1`] = `
 
 <InputField
   className="emotion-0"
+  hint={null}
```

Comment:
Not sure how this got into the snapshots but looks like a mistake (as the code change does not effect this file) - if you could revert this change that would be awessssoommmeee.